Library/Formula/consul.rb

```diff
@@ -18,159 +18,172 @@ class Consul < Formula
 
   depends_on "go" => :build
 
+  go_resource "github.com/DataDog/datadog-go" do
+      url "https://github.com/DataDog/datadog-go.git",
```

Comment:
You're indenting too hard here, and in each resource below. Should be two spaces, not four.